operator/hack/addons.yaml

```diff
@@ -2,28 +2,123 @@
 # within a cluster. logcli pod helps write queries, promtail writes logs, etc
 ---
 apiVersion: v1
-kind: Pod
+kind: ServiceAccount
+metadata:
+  name: logcli-loki
+  labels:
+    app.kubernetes.io/name: logcli
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: promtail-loki
+  labels:
+    app.kubernetes.io/name: promtail
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logcli-loki-clusterrole
+  labels:
+    app.kubernetes.io/name: logcli
+rules:
+- apiGroups:
+  - loki.grafana.com
+  resources:
+  - application
+  resourceNames:
+  - logs
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: promtail-loki-clusterrole
+  labels:
+    app.kubernetes.io/name: promtail
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - watch
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
 metadata:
-  name: logcli
-  namespace: loki
+  name: logcli-loki-clusterrolebinding
+  labels:
+    app.kubernetes.io/name: logcli
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: logcli-loki-clusterrole
+subjects:
+- kind: ServiceAccount
+  name: logcli-loki
+  namespace: openshift-logging
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: promtail-loki-clusterrolebinding
+  labels:
+    app.kubernetes.io/name: promtail
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: promtail-loki-clusterrole
+subjects:
+- kind: ServiceAccount
+  name: promtail-loki
+  namespace: openshift-logging
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: logcli-loki
   labels:
     app.kubernetes.io/name: logcli
 spec:
-  containers:
-  - name: logcli
-    image: docker.io/grafana/logcli:2.2.0-amd64
-    env:
-      - name: LOKI_ADDR
-        value: http://loki-querier-http-lokistack-sample.loki.svc.cluster.local:3100
-    command: [ "/bin/sh", "-c", "--" ]
-    args: [ "while true; do sleep 30; done;" ]
-
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: logcli
+  template:
+    metadata:
+      name: loki-logcli
+      labels:
+        app.kubernetes.io/name: logcli
+    spec:
+      containers:
+        - name: logcli
+          image: docker.io/grafana/logcli:2.4.1-amd64
+          imagePullPolicy: IfNotPresent
+          command:
+            - /bin/sh
+          env:
+            - name: LOKI_ORG_ID
+              value: application
+            - name: LOKI_ADDR
+              value: http://lokistack-gateway-http-lokistack-dev.openshift-logging.svc:8080/api/logs/v1/application
```

Comment:
With #5607 the new svc is `lokistack-dev-gateway-http` instead of `lokistack-gateway-http-lokistack-dev`